MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -131,7 +136,8 @@ public FontDescriptionProcessor()
                 }
 			}
 			catch(Exception ex) {
-				context.Logger.LogImportantMessage("{0}", ex.ToString());
+				//context.Logger.LogImportantMessage("{0}", ex.ToString());

```

Comment:
Commenting this call i can convert the output to the pipeline from:
![before](https://cloud.githubusercontent.com/assets/11319966/13192705/4e0edcfc-d76f-11e5-8d12-c8a5db546d2c.PNG)
To:
![after](https://cloud.githubusercontent.com/assets/11319966/13192708/52b0fe7a-d76f-11e5-9b0c-19cca655a8db.PNG)
Before merging if there nothing against this change i'll remove the try catch completely